README.md

```diff
@@ -137,7 +137,7 @@ struct MyRow {
     name: String,
 }
 
-let mut insert = client.insert("some")?;
+let mut insert = client.insert::<MyRow>("some").await?;
```

Comment:
Instead of `` ```rust,ignore `` we could use `` ```rust,norun `` so that these at least get compile-tested.

I could take that on as a task.

Reply:
Can be done as a part of finalizing https://github.com/ClickHouse/clickhouse-rs/pull/160